benchmarkapps/BasicApi/Controllers/PetController.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using BasicApi.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace BasicApi.Controllers
+{
+    [Authorize("pet-store-reader")]
+    [Route("/pet")]
+    public class PetController : ControllerBase
+    {
+        public PetController(BasicApiContext dbContext)
+        {
+            DbContext = dbContext;
+        }
+
+        public BasicApiContext DbContext { get; }
+
+        [HttpGet("{id}", Name = "FindPetById")]
+        public async Task<IActionResult> FindById(int id)
```

Comment:
Should this be `ActionResult<Pet>`? It's what we'd tell users to use.

Reply:
@rynowak before I check this PR, any of @pranavkm's suggested changes you think I should make i.e. are we fine as-is w/ these apps?

Reply:
I think it would be good to hew to what we're telling customers to do - if it's mot too much trouble